Cargo.toml

```diff
@@ -42,6 +43,7 @@ libloading = "0.8"
 [target.'cfg(target_family = "windows")'.dependencies]
 windows-sys = { version = "0.59", features = [
     "Win32_Foundation",
+    "Win32_Security",
```

Comment:
Are you sure we need this? This is for Miri itself calling Windows API functions, which we only do in one place:

https://github.com/rust-lang/miri/blob/b63d8fcbda5a2617f2cb351f54668a55a14fa344/src/shims/unix/fs.rs#L211-L216

I don't know why "System_IO" is even in this list, we don't seem to use that.